x-pack/test/functional/apps/ml/anomaly_detection/anomaly_explorer.ts

```diff
@@ -64,7 +64,7 @@ export default function ({ getService }: FtrProviderContext) {
   const elasticChart = getService('elasticChart');
 
   describe('anomaly explorer', function () {
-    this.tags(['mlqa']);
+    this.tags(['mlqa', 'dima']);
```

Comment:
tag accidentally left in.

Reply:
removed in 95171b2a377488c53244ec1c4f9e31a38b366c4c